pyproject.toml

```diff
@@ -36,11 +38,11 @@ dependencies = [
     # less than or equal to Python 3.10. In order to ensure we have
     # a consistent dependency closure across all Python versions,
     # we explicitly include ruamel.yaml.clib as a dependency.
-    "ruamel.yaml.clib>=0.2.0,<=0.2.7",
+    "ruamel.yaml.clib>=0.2.0,<=0.2.8",
     "prompt-toolkit>=3.0.24,<3.0.39",
     "distro>=1.5.0,<1.9.0",
     "awscrt>=0.19.18,<=0.21.2",
-    "python-dateutil>=2.1,<=2.8.2",
+    "python-dateutil>=2.1,<=2.9.0",
```

Comment:
Why do we need to increase the ceiling?

Reply:
2.9.0 removes a call to a deprecated function